lnst/Recipes/ENRT/BasePvPRecipe.py

```diff
@@ -53,6 +53,9 @@ class BasePvPRecipe(PingTestAndEvaluate, PerfRecipe):
     """
 
     driver = StrParam(mandatory=True)
+    nic_speed = StrParam()
+    nic_model = StrParam()
```

Comment:
this should now be removed since we have the base common class for these parameters

Reply:
Good catch, fixed.